src/main/scala/is/hail/methods/SplitMulti.scala

```diff
@@ -37,12 +39,140 @@ class ExprAnnotator(val ec: EvalContext, t: TStruct, expr: String, head: Option[
   }
 }
 
-class SplitMultiPartitionContext(
+class SplitMultiRowIR(rowIRs: Array[(String, IR)], entryIR: Array[(String, IR)], oldMatrixType: MatrixType) {
+  val oldRowIR = Ref("va")
```

Comment:
the old row should not be called "va", "va" traditionally refers to the row values only (without the entries).